requirements/requirements.in

```diff
@@ -88,7 +88,7 @@ PyYAML~=5.1
 pyzxcvbn==0.8.0
 qrcode==4.0.4
 quickcache~=0.5
-redis==3.2.1
+redis==3.3.0
```

Comment:
I agree with Danny https://github.com/dimagi/commcare-hq/pull/28550#discussion_r495119804, we should upgrade to whatever the latest version of 3.3.x is (apparently 3.3.11).

Reply:
@millerdev i feel reluctant to do that as this has already gone thru QA and this comment came in when I was offline. I would want to QA again even bumping from 0 to 11, but I think we should fix the issue with intermittent failures first. Happy to go back and upgrade even further—maybe whatever the latest version is that we can support given our other packages—and then put that through QA